src/utils.jl

```diff
@@ -66,12 +65,16 @@ function xy_to_offset(fi::FileInfo, pos::Position)
     end
     return b + line_b
 end
+xy_to_offset(fi::FileInfo, pos::Position) = xy_to_offset(fi.parsed_stream.textbuf, pos)
 
 """
 Convert a 1-based byte offset to a 0-based line and character number
 """
-function offset_to_xy(fi::FileInfo, b::Integer)
-    sf = JuliaSyntax.SourceFile(fi.parsed_stream)
+function offset_to_xy(ps::Union{AbstractString, JuliaSyntax.ParseStream}, b::Integer)
+    sf = JuliaSyntax.SourceFile(ps)
```

Comment:
```suggestion
    sf = JS.SourceFile(ps)
```
Let's define
```julia
using JuliaSyntax as JS
using JuliaLowering as JL
```
within JETLS.jl and use these aliases consistently across the codebase.